specs/phase1/beacon-chain.md

```diff
@@ -320,7 +328,7 @@ class ShardSignableHeader(Container):
 class ShardState(Container):
     slot: Slot
     gasprice: Gwei
-    data: Bytes32
+    transition_digest: Bytes32
```

Comment:
Should `transition_digest` eventually be `Root`?  
Looks like `compute_shard_transition_digest` will use ssz hash tree root for the whole digest

Reply:
Yeah, Root will be the appropriate type